### PR TITLE
[kf5bookmarks] new port

### PR DESCRIPTION
**Pull request by wrobelda**
**Describe the pull request**
  Adds new kf5bookmarks port. 
  Depends on:
  - [x] #19766. It was merged to this PR for testing purposes, but please review separately. I will rebase once it gets merged upstream.
  - [x] #19439. It was merged to this PR for testing purposes, but please review separately. I will rebase once it gets merged upstream. 
  - [x] #19756. It was merged to this PR for testing purposes, but please review separately. I will rebase once it gets merged upstream. 

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  All official, plus `arm64-osx`

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

### CODE REVIEW

**Comment by wrobelda** [comment]
@NancyLi1013 the CI infra seems to have some random problems, look like one of the agents has leftovers in C:\installed

**Comment by wrobelda** [comment]
@NancyLi1013 this no longer has any dependencies, ready to be merged.

File: ports/kf5bookmarks/fix_config_cmake.patch [hunk 1]
```diff
@@ -0,0 +1,34 @@
+From 459cfb91027b27831caaa63584859f51af88d778 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Dawid=20Wro=CC=81bel?= <me@dawidwrobel.com>
+Date: Mon, 30 Aug 2021 23:02:11 +0200
+Subject: [PATCH] Add support for static builds
+
+---
+ KF5BookmarksConfig.cmake.in | 11 +++++++++++
+ 1 file changed, 11 insertions(+)
+
```
Comment (NancyLi1013) [inline ports/kf5bookmarks/fix_config_cmake.patch:9]: Could you please remove these information?

File: ports/kf5bookmarks/vcpkg.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "kf5bookmarks",
+  "version-semver": "5.84.0",
```
Comment (NancyLi1013) [inline ports/kf5bookmarks/vcpkg.json:3]:
```suggestion
  "version": "5.84.0",
```
This should be version here.

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: ports/kf5bookmarks/fix_config_cmake.patch (hunk 1 above)
Comment (wrobelda) [reply ports/kf5bookmarks/fix_config_cmake.patch]: Done.

File: ports/kf5bookmarks/vcpkg.json (hunk 2 above)
Comment (wrobelda) [reply ports/kf5bookmarks/vcpkg.json]: Done.

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
LGTM now. thanks for adding this port. @wrobelda

**Comment by BillyONeal** [comment]
Thanks for the new port!

**Pull request merged**